torchaudio/functional.py

```diff
@@ -169,6 +169,9 @@ def spectrogram(
         spec_f /= window.pow(2.).sum().sqrt()
     if power is not None:
         spec_f = complex_norm(spec_f, power=power)
```

Comment:
```
# spec_f = complex_norm(spec_f, power=power)
```
Did you forget to comment out the original?

Reply:
Thanks! I just updated it.